bbot/modules/lightfuzz/lightfuzz.py

```diff
@@ -167,8 +170,13 @@ async def finish(self):
             except InteractshError as e:
                 self.debug(f"Error in interact.sh: {e}")
 
-    # If we've disabled fuzzing POST parameters, back out of POSTPARAM WEB_PARAMETER events as quickly as possible
     async def filter_event(self, event):
+        # Unless configured specifically to do so, avoid running against confirmed WAFs
+        if self.avoid_wafs and any(tag in ["cdn-cloudflare", "cdn-akamai", "cdn-incapsula"] for tag in event.tags):
```

Comment:
If we need to track which cloud providers are WAFs, this should be done in cloudcheck

Reply:
Cloudcheck is about to get rustified so now is a good time to add it

Reply:
you ok with going ahead with this temporarily until that's done?